test/redisGraphAPITest.js

```diff
@@ -455,4 +455,44 @@ describe("RedisGraphAPI Test", () => {
 			);
 		}
 	});
+
+	it("testOptionalMatch", async () => {
```

Comment:
@DvirDukhan can you please also add test for "Left outer join"? meaning mixed null and not null values in the same record

Reply:
there is one
`MATCH (a) OPTIONAL MATCH (a)-[e]->(b) RETURN a, e, b ORDER BY ID(a)`

Reply:
Sorry I missed `assert.notEqual(record.get(0), null);`, why not really check the value?